src/main/java/complete_task_use_case/CompleteTaskController.java

```diff
@@ -0,0 +1,8 @@
+package complete_task_use_case;
+
+public class CompleteTaskController {
+
+    public CompleteTaskOutputData complete(int dayId, String title) {
```

Comment:
Please change dayId to dayIndex.... this is the current method signature "completeTask(String day, String title)"

Reply:
Will do that.